examples/sql-integration/tests/cli.rs

```diff
@@ -59,7 +59,7 @@ fn simulate() {
     thread::spawn(move || {
         while let Some(Ok(line)) = stdout.next() {
             println!("party0> {line}");
-            if line.contains("MPC Output: 1 rows") {
+            if line.contains("MPC Output: 10 rows") {
```

Comment:
It would be nice if in the test, we would actually open the output db and check that we have written the expected data.
However, I think some of the other examples currently also don't check that the computed output is correct. We can also track this in an issue and do at a later time.